src/firefly/js/visualize/ImageViewerLayout.jsx

```diff
@@ -493,7 +493,7 @@ function makeMessageArea(pv,plotShowing,onScreen, sizeViewable, loadingRawData)
     if (pv.serverCall==='success') {
         if (loadingRawData) {
             return (
-                <ImageViewerStatus message={'Loading Color Data'} working={true}
+                <ImageViewerStatus message={'Loading Image Data'} working={true}
```

Comment:
As I mentioned on the [FIREFLY-890 ticket](https://jira.ipac.caltech.edu/browse/FIREFLY-890), I think (if I've understood the issue correctly) "Loading image rendering" would be both clearer and more likely to be reported back accurately by a user if the distinction between this and the loading-image-from-source phase were ever to be an issue.

Reply:
I like this idea.  I will change it to `Loading image rendering`